### PR TITLE
DEV021 create html forms for user model

### PR DESCRIPTION
**Pull request by MariaPaharelava**
Create html forms for user model
[card](https://trello.com/c/UooMXmKh)

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
+  <%= stylesheet_link_tag 'main_application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+</head>
 
-  <body>
-    <% if user_signed_in? %>
-      <%= react_component("shared/navigation/navigation", {image: current_user.image, id: current_user.id}) %>
-    <% else %>
-      <%= react_component("shared/navigation/navigation") %>
+<>
```
Comment (Maxxx911) [inline app/views/layouts/application.html.erb]: Redundant brackets

File: app/javascript/components/shared/navigation/navigation.jsx [hunk 2]
```diff
@@ -70,7 +66,7 @@ class Navigation extends React.Component {
                       Me
                     </a>,
                     <a
-                      href={"/"}
+                      href={"/users/" + this.props.id + "/edit"}
```
Comment (Maxxx911) [inline app/javascript/components/shared/navigation/navigation.jsx]: please use `/users/${this.props.id}/edit
`

File: app/views/users/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  <% end %>
+
+</div>
+
+<script>
+  function readURL(input) {
+    console.log(input);
+    if (input.files && input.files[0]) {
+      const reader = new FileReader();
+      console.log(input);
```
Comment (Maxxx911) [inline app/views/users/edit.html.erb]: Delete console.log

File: app/views/users/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    <%= f.text_field :nickname, placeholder: "Nickname...", class: "form-control", autocomplete: @user.nickname %><br/>
+    <%= f.submit "Sign in", class: "btn btn-primary" %>
+  <% end %>
+
+</div>
+
+<script>
+  function readURL(input) {
+    console.log(input);
```
Comment (Maxxx911) [inline app/views/users/edit.html.erb]: Delete console.log

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -10,6 +10,7 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
+ *= require font-awesome
+*= require material/material-kit.css
```
Comment (longbeardmen) [inline app/assets/stylesheets/application.css]: missing space

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -27,19 +27,61 @@ def create
...
-  def update
+  def edit
+    @genres = Genre.all
+    @musician_skill = MusicianSkill.all
+    @user = current_user
+    @id = current_user.id
+    @image = current_user.image
   end
 
-  def destroy
```
Comment (longbeardmen) [inline app/controllers/users_controller.rb]: how to remove account without this action?

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -27,19 +27,61 @@ def create
...
+
+  def update_user(user)
+    current_user.update(
+      first_name: user[:first_name],
+      second_name: user[:second_name],
+      nickname: user[:nickname]
+    )
+  end
+
+  def create_photo(user)
```
Comment (longbeardmen) [inline app/controllers/users_controller.rb]: why this is places in `UsersController`?
why should we destroy image?

File: app/controllers/users_controller.rb [hunk 8]
```diff
@@ -27,19 +27,61 @@ def create
...
+
+  def update_genres(user)
+    user[:genres].split(',').each do |item|
+      current_user.genres << Genre.find(item)
+    end
+  end
+
+  def update_musician_skills(user)
+    user[:musician_skills].split(',').each do |item|
+      current_user.musician_skills << MusicianSkill.find(item.to_i)
```
Comment (longbeardmen) [inline app/controllers/users_controller.rb]: `.to_i` is redundant. `.find` works fine with numeric strings

File: app/controllers/users_controller.rb [hunk 9]
```diff
@@ -27,19 +27,61 @@ def create
...
+    end
+  end
+
+  def update_user(user)
```
Comment (longbeardmen) [inline app/controllers/users_controller.rb]: this method name is confusing. it's not super different in comparison `update`, but it updates only users names

File: app/controllers/users_controller.rb (hunk 8 above)
Comment (longbeardmen) [reply app/controllers/users_controller.rb]: like here
https://github.com/rubizza-camp/uber-music/pull/65/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R60

File: app/controllers/users_controller.rb (hunk 6 above)
Comment (MariaPaharelava) [reply app/controllers/users_controller.rb]: Devise

**Pull request merged**